src/machine.rs

```diff
@@ -420,6 +420,81 @@ impl<'tcx> PrimitiveLayouts<'tcx> {
     }
 }
 
+/// A thread's CPU affinity mask determines the set of CPUs on which it is eligible to run.
```

Comment:
Please move this to another file, `machine.rs` is already way too big. Since it is related to threads, a new file in `src/concurrency` is probably a good choice.